Kenttrann2302/StudyHub-Backend
# Fix requirements txt and format files

# Why was this done?

To ensure consistent code format and style for readability. To remove unnecessary code from the repository. Main goal is just overall readability.

# What does this PR do?

- This won't be continue until [this PR](https://github.com/Kenttrann2302/StudyHub-Backend/pull/14) is merged in
- Format the files with `black`
- Sort the imports
- Remove unused imports
- Remove excessive dependencies from `requirements.txt`
- Remove excessive dependencies for AWS Lambda functions
  - We will investigate ways to streamline dependency installation later

# How should this be tested?

- Create a new virtual environment with `venv`, ensure you name it to something different than the virtual environment you already have
- Activate the virtual environment
- Run `python install -r requirements.txt` from the root directory of the project (note how we only need to specify `python` here since we have activated the virtual environment
- Rerun the registration flow and ensure things didn't break

# Related issues

- This blocks:
  - #17 
  - #18 
  - #19 

@Kenttrann2302
